Makefile

```diff
@@ -14,4 +14,4 @@ setup:
 	ADMIN=on bundle install
 
 docker:
-	docker run --rm -p 4000:4000 -v `pwd`:"/srv/jekyll" jekyll/jekyll:3.7.3 bash -c "bundle install; jekyll serve --drafts"
+	docker run --name layer5 --rm -p 4000:4000 -v `pwd`:"/srv/jekyll" jekyll/jekyll:3.7.3 bash -c "bundle install; jekyll serve --drafts"
```

Comment:
Should this be `meshery` instead?

Reply:
Good question. This docker command is for running the layer5 site, hence layer5 was my thought.

Reply:
Oops, I just realized that this was running `jekyll`.